service/turn.rest/pom.xml

```diff
@@ -123,6 +145,68 @@
           </execution>
         </executions>
       </plugin>
+
```

Comment:
You are right, I will remove this part from the pom. It was intended for turn.proxy I think.